### ISSUE TITLE
HedgeModManager opens and crashes

### ISSUE THREAD

**Issue by TheShovel** [sonic2kk/steamtinkerlaunch#706]
## System Information
- SteamTinkerLaunch version: 12.0
- Distribution: Manjaro KDE
- Installation Method: Package manager

## Issue Description
HedgeModManager takes a lot to configure, and then opens a window that freezes, and crashes shortly after.

## Logs
```
[dumbass@ProfesionalDumbass ~]$ steamtinkerlaunch hmm start nightly

(yad:91455): Gtk-WARNING **: 08:43:27.632: Theme parsing error: gtk.css:73:46: The style property GtkScrolledWindow:scrollbars-within-bevel is deprecated and shouldn't be used anymore. It will be removed in a future version
Checking latest available HedgeModManager Development version
Latest HedgeModManager is already downloaded or was downloaded previously
Found up-to-date HedgeModManager executable - Nothing to download
Finished installing HedgeModManager
Adding HedgeModManager to application menu and setting up link handlers
Setting up installed HedgeModManager compatible games
Error calling StartServiceByName for org.freedesktop.Notifications: Timeout was reached
Error calling StartServiceByName for org.freedesktop.Notifications: Timeout was reached
Running configuration for 'Sonic Frontiers'
Finished configuring installed HedgeModManager games
Starting HedgeModManager
Error calling StartServiceByName for org.freedesktop.Notifications: Timeout was reached

```

[steamtinkerlaunch.log](https://github.com/sonic2kk/steamtinkerlaunch/files/10349957/steamtinkerlaunch.log)

**Comment by TheShovel** [comment]
Fixed it by running it with a older version of proton.

**Comment by sonic2kk** [comment]
All good, this is documented on the wiki already (which you may have seen since you fixed it by presumably using Proton 5.0 :-))
